docs/content/docs/ci/Logging.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Logging
+type: docs
+---
+
+# Logging
+
+`com.avito.logger.Logger` used both in Gradle and in Android Runtime
+
+to get logger instance use one of `loggerFactory.create(className)`
```

Comment:
```suggestion
```
It's not clear what is `loggerFactory` here.
I think it can be omitted because below we have exact examples.